### DISCUSSION TITLE
如何实现回车将输入的字母上屏

### DISCUSSION THREAD

**Discussion by imbamboo** [KyleBing/rime-wubi86-jidian#93] [Q&A] [answered]
假设我现在处于五笔输入法状态，我随便找了N个字母，可能有候选词，也可能没有。
有没有办法设置，敲回车时，直接把刚才输入的字母上屏，不用匹配候选词（不管有没有匹配到）？

**Comment by imbamboo** [comment]
https://github.com/rime/home/issues/28
这里看到一个“Shift定向Return”的QA，貌似也符合需求，但使用上感觉怪怪的。
有没有更恰当的做法？

**Reply by imbamboo** [reply]
这个虽然把输入的字母上屏了，但同时，也切入的英文输入法状态了（要输入中文，得再按一次shift切回来）。
我不希望是这样。
我的本意只是临时输入几个英文字母，而不是切换去英文输入状态。

**Reply by KyleBing** [reply, marked as answer]
https://github.com/KyleBing/rime-wubi86-jidian#2-%E5%9B%9E%E8%BD%A6%E6%B8%85%E7%A0%81
